types/ismobilejs/index.d.ts

```diff
@@ -3,41 +3,55 @@
 // Definitions by: Maksim Karelov <https://github.com/Ty3uK>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export const any: boolean;
-export const phone: boolean;
-export const tablet: boolean;
-export const seven_inch: boolean;
-
-export const apple: {
-    phone: boolean;
-    ipod: boolean;
-    tablet: boolean;
-    device: boolean;
-};
-
-export const android: {
-    phone: boolean;
-    tablet: boolean;
-    device: boolean;
-};
-
-export const amazon: {
-    phone: boolean;
-    tablet: boolean;
-    device: boolean;
-};
-
-export const windows: {
-    phone: boolean;
-    tablet: boolean;
-    device: boolean;
-};
-
-export const other: {
-    blackberry_10: boolean;
-    blackberry: boolean;
-    opera: boolean;
-    firefox: boolean;
-    chrome: boolean;
-    device: boolean;
-};
+interface Apple {
+  phone: boolean;
+  ipod: boolean;
+  tablet: boolean;
+  device: boolean;
+}
+
+interface Android {
+  phone: boolean;
+  tablet: boolean;
+  device: boolean;
+}
+
+interface Amazon {
+  phone: boolean;
+  tablet: boolean;
+  device: boolean;
+}
+
+interface Windows {
+  phone: boolean;
+  tablet: boolean;
+  device: boolean;
+}
+
+interface Other {
+  blackberry_10: boolean;
+  blackberry: boolean;
+  opera: boolean;
+  firefox: boolean;
+  chrome: boolean;
+  device: boolean;
+}
+
+interface IsMobile {
+  any: boolean;
+  phone: boolean;
+  tablet: boolean;
+  seven_inch: boolean;
+
+  apple: Apple;
```

Comment:
`require("ismobilejs").apple` is `undefined` when I tried it, but would compile if used like this. Maybe the `IsMobile` interface doesn't represent the module itself, but the result of calling it:
```ts
interface IsMobile { ... }
declare function isMobile(userAgent?: string): IsMobile;
```

Reply:
This is for browser compatible. In node we must use `isMobile(userAgent)`, but in browser (using AMD, browserify, or in `window`) library instantiate isMobile class

Reply:
OK, could you add a comment around the browser-only declarations?

Reply:
It's a little problematic, because browser and node-package properties are included in one `IsMobile` interface.